app.js

```diff
@@ -14,12 +14,44 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials(__dirname + "/views/partials")
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then((beersData)=>{
+      console.log('Beer data fetched')
+      res.render('beers', {beersData});
+    })
+    .catch((err)=>{console.log(err)})
+});
+app.get('/random-beer', (req, res) => {
+  punkAPI.getRandom()
+    .then((beer)=>{
+      console.log('rng beer passed')
+      console.log(beer)
+      res.render('random-beer', {beer});
+    })
+    .catch((err)=>console.log(err))
+});
+//Trying to make a unique beer page
+
+app.get('/beer/*', (req, res) => {
+    let beerId = req.params[0]
```

Comment:
best practice for using params is to declare the name of the param key inside the url and then use destructure to obtain it.
````
app.get('/beer/:beerId', (req, res) => {
    let { beerId } = req.params
````